susemanager-utils/testing/automation/backend-unittest-pgsql.sh

```diff
@@ -1,7 +1,25 @@
 #! /bin/sh
+if [ -z ${PRODUCT+x} ];then
```

Comment:
What is this doing? Setting the environment variable if it doesn't exist? Why the `+x`?

Reply:
https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash